.gitignore

```diff
@@ -13,3 +13,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.idea
```

Comment:
🤔 this look weird? if we have a reason to keep it please share the explanation. If not, please remove this line :)

Reply:
Forget it, I remember that you are using webstorm as IDE, so ignored this comment.